BullsAndCows/BullsAndCowsGame.cs

```diff
@@ -24,12 +24,21 @@ public BullsAndCowsGame(SecretGenerator secretGenerator)
             // case 5: some of correct digits with correct locations, some with wrong loc, "1345"-"1A2B", except for "3A1B"
             // case 6: correct digits, but some are correct loc, some are not, "1432"-"2A2B"
             // case 6: all wrong, "5678"-"0A0B"
+            int bulls = 0;
             if (guess.Equals(secret))
             {
                 return "4A0B";
             }
 
-            return string.Empty;
+            for (int i = 0; i < secret.Length; i++)
+            {
+                if (guess[i] == secret[i]) // same position
```

Comment:
One line comment should be commented above the code, like: "//(space)this is a comment"